src/components/Search/components/DeliveryButton/DeliveryButton.jsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import Button from "../../../Button/Button";
+
+export default function DeliveryButton() {
+  return (
+    <Button
+      variant="contained"
+      sx={{
+        ml: "1rem",
+        textTransform: "none",
+        width: 132,
+        backgroundColor: "#307460",
+        padding: "8px 16px",
+      }}
+    >
+      New Delivery
+    </Button>
+  );
```

Comment:
For some reason this button is blue on hover. Maybe it should be the same green color but darkened

Reply:
ok well.